api/src/main/java/org/openmrs/Obs.java

```diff
@@ -1080,11 +1080,31 @@ public String getValueAsString(Locale locale) {
 		return "";
 	}
 	
-	private static DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+	private static ThreadLocal<DateFormat> dateFormat = new ThreadLocal<DateFormat>() { // Made DateFormat Threadsafe using ThreadLocal		

```

Comment:
Found tests at http://www.javacodegeeks.com/2010/07/java-best-practices-dateformat-in.html